README.md

```diff
@@ -12,6 +12,7 @@ Launch, machine and parameter files required to bringup the HERO robot
 - Back-Up of virtualbox image which are located in `~/vbox_images`:
   - Stop systemd service: `sudo systemctl stop hero1-speech5-windows-speech.service` or `sudo systemctl stop hero1-demo-speech5-windows-speech.service`
   - Actual back-up i.e.: `scp -r ~/vbox_images amigo@hero2.local:~/vbox_images_backup`
+- Back-Up the odometer files: `scp -r ~/odometer amigo@hero2.local:~/odometer_backup`
```

Comment:
does this store the odometer file on hero2? Why store the backup there?
Could you explain how this backup system works for me.

Reply:
Store is temporarily